infra/cifuzz/cifuzz.py

```diff
@@ -185,10 +192,13 @@ def run_fuzzers(fuzz_seconds, workspace):
     return False, False
   fuzz_seconds_per_target = fuzz_seconds // len(fuzzer_paths)
 
+  old_build_path = download_old_build_dir(project_name, out_dir)
```

Comment:
nit: let's call it `ossfuzz_build_path` instead of `old_build_path`, to be more explicit

Reply:
Done.